### DISCUSSION TITLE
Play JSON 2.10.2 and 3.0.0 releases in mvnrepository?

### DISCUSSION THREAD

**Discussion by kkeinanen** [playframework/playframework#12064] [General]
Hi,

Scala Steward notified me about play-json 3.0.0 being available in [MVNRepository](https://mvnrepository.com/artifact/org.playframework/play-json_2.13/3.0.0).  I have been trying to find release notes, but the latest release I see in [Github](https://github.com/playframework/play-json/releases is 2.10.1) is 2.10.1.

Are there source repositories I am unaware of, or are these illegit or unintentional binary releases?

Thanks,

- Kai

**Comment by mkurz** [comment]
These are intentional releases, announcement will come soon. play-json 3.0.0 changed the groupId to `org.playframework` but should otherwise be identical to play-json 2.10.x.

**Reply by kkeinanen** [reply]
Thanks!
